### DISCUSSION TITLE
Problems Trying To Use The Modern Theme

### DISCUSSION THREAD

**Discussion by mkniller** [dotnet/docfx#9509] [General]
Hello braintrust, I am struggling to get the modern theme working in my DocFX project.  I ran the docfx upgrade command and then did the docfx init.  From there I did the following:

![image](https://github.com/dotnet/docfx/assets/8039354/db6307c9-a99d-4c8d-abdd-914e23d9ef64)

From the list shown, the Modern theme is included as an embedded theme, so I downloaded the DocFX repo and copied the data in the templates/modern folder into a templates folder I created on my system.

When the page loads it looks like this:

![image](https://github.com/dotnet/docfx/assets/8039354/f79cdff1-d867-43fe-9c24-52747dca3c73)

Can anybody offer some advice on how you got the modern theme working?

**Comment by VaclavElias** [comment]
I am not sure if the `modern` template is intended to work with old docfx projects. Or is it a completely new project?

You can have a look at my example which is working well, with the `modern` template, but it is a new project.

Source: https://github.com/stride3d/stride-community-toolkit/tree/main/docs
Result: https://stride3d.github.io/stride-community-toolkit/index.html

**Comment by filzrev** [comment]
It seems `docfx` command is resolved to old docfx version.
What's results shown when executing following commands.

> `docfx --version`

It should be resolved to `2.74.1`.

> `which docfx` or  `where docfx` (Windows)

if docfx is installed as .NET global tools.
It' expected to be `%USERPROFILE%\.dotnet\tools\docfx.exe`.
